ui2/src/pages/common.ts

```diff
@@ -1,5 +1,38 @@
 import { CreateEvent } from '@daml/ledger';
+import { DropdownProps, DropdownItemProps } from 'semantic-ui-react';
 
 export type ServicePageProps<T extends object> = {
   services: Readonly<CreateEvent<T, any, any>[]>;
 };
+
+export function isStringArray(strArr: any): strArr is string[] {
```

Comment:
My Daml mind rebels against this. Is there a way to refactor the code so you're not passing `any` around, to avoid using this function?

Reply:
This was copied over in `ui`, to have a select dropdown that can also function as a regular text input...not sure it's actually necessary with the type of dropdowns I'm using it for, so I can take a look at getting rid of it.

I also don't enjoy seeing `any`, but using it for a function that checks what a type is (such as `Array.isArray`) does make sense to me - especially because it's use here is to make dealing with `DropdownProps` more "type safe", since `DropdownProps` is `[key: string]: any` which there's not much you can do about.